### DISCUSSION TITLE
'micro' or custom installation

### DISCUSSION THREAD

**Discussion by bakeiro** [osm-search/Nominatim#3370] [Ideas]
**Is your feature request related to a problem? Please describe.**
I was thinking about installing nominatim in my server to retrieve countries geometries and with that info use it in my software, thing is that this needs to much memory and so much resources, and I just need the countries geometries, so I feel that maybe is a bit overkill for me...

**Describe the solution you'd like**
I would like to have a 'custom installation' so I just install what I need (countries geometries) and the rest of the stuff I don't needed it

**Describe alternatives you've considered**
I considered download the geometries of countries somewhere else, but feels like a longer process to find this data

**Comment by mtmail** [comment]
Are you planning to use Nominatim as a search engine (Nominatim's main use case)? Or only to import geometries?

**Comment by bakeiro** [comment]
in principle for only geometries (maybe the search engine as well but that not sure)

**Comment by mtmail** [comment]
Installing Nominatim indeed seems overkill for your project. I'd ask on https://community.openstreetmap.org/ for suggestions which approach/software/tools/data to use.

You could also install Nominatim, import your required data and then export only the data you need into a new database.
